src/custom/components/Menu/index.tsx

```diff
@@ -343,20 +373,15 @@ export function Menu({ darkMode, toggleDarkMode, isClaimPage, handleOnClickClaim
           )}
         </MenuItemResponsive>
 
-        <Separator />
-
         <Policy to="/terms-and-conditions" onClick={close} onKeyDown={close}>
           Terms and conditions
         </Policy>
         {/* 
         <Policy to="/privacy-policy">Privacy policy</Policy>
         <Policy to="/cookie-policy">Cookie policy</Policy> 
         */}
-        {showVCOWClaimOption && (
```

Comment:
shouldn't we keep these commented to stay similar to original? or is this all custom

Reply:
The mod file `src/custom/components/Menu/MenuMod.tsx` still references the original code (commented out). So should be good to not have it in our custom `index` file?